pymongo/asynchronous/encryption.py

```diff
@@ -516,6 +516,11 @@ class Algorithm(str, enum.Enum):
 
     .. versionadded:: 4.4
     """
+    TEXTPREVIEW = "TextPreview"
```

Comment:
Are we required to call this `TEXTPREVIEW` instead of calling it `TEXT` and designating "feature preview"?

Reply:
Yes, following the pattern we used for RANGEPREVIEW, to ensure that if the semantics change it won't do so silently.